bundles/org.eclipse.orion.client.ui/web/orion/webui/littlelib.js

```diff
@@ -603,6 +603,20 @@ define(["orion/util"], function(util) {
 		return newNodes;
 	}
 
+  /**
+   * @description This function is a delegate to set the attribute on a given node

```

Comment:
This function is a delegate to set the **value of a named** attribute on a given node